scalafmt-core/shared/src/main/scala/org/scalafmt/util/TokenOps.scala

```diff
@@ -222,4 +222,14 @@ object TokenOps {
     }
   }
 
+  // For using to be the soft kw it also has to be preceded by paren
+  def isRightImplicitOrUsingSoftKw(
+      ft: FormatToken,
+      soft: SoftKeywordClasses
+  ): Boolean = ft match {
+    case FormatToken(_: LeftParen, soft.KwUsing(), _) => true
```

Comment:
there is only one problem with this approach: this will not be recognized (and we need to add a test, too)
```
def foo( /* go scala3 */ using ... )
```
you might need to move this method into `FormatOps` instead and implement something like this:
```
  def isLeftImplicitOrUsingSoftKw(
      ft: FormatToken,
      soft: SoftKeywordClasses
  ): Boolean = ft.left match {
    case _: KwImplicit => true
    case soft.KwUsing() => prevNonComment(prev(ft)).is[LeftParen]
    case _ => false
  }
```

Reply:
Speaking of `ApplyUsing` there is no similar counterpart for the definition so I would stay with the paren check.